src/System.Management.Automation/engine/parser/token.cs

```diff
@@ -1119,17 +1124,19 @@ public static class TokenTraits
             /*              Command */ "command",
             /*               Hidden */ "hidden",
             /*                 Base */ "base",
+            /*            AtAtCurly */ "@@splat",
 
             #endregion Text for keywords
         };
 
 #if DEBUG
         static TokenTraits()
         {
-            Diagnostics.Assert(s_staticTokenFlags.Length == ((int)TokenKind.Base + 1),
-                               "Table size out of sync with enum - _staticTokenFlags");
-            Diagnostics.Assert(s_tokenText.Length == ((int)TokenKind.Base + 1),
-                               "Table size out of sync with enum - _tokenText");
+            var maximumEnumValueOfTokenKind = Enum.GetValues(typeof(TokenKind)).Cast<int>().Max();
```

Comment:
In general I would agree but because the statements are already inside a `#if DEBUG` pragma (see 3 lines above), the `static TokenTraits()` method is only executed in Debug builds anyway